src/transport/SessionHolder.h

```diff
@@ -41,12 +41,7 @@ class SessionHolder : public SessionReleaseDelegate
     SessionHolder(SessionHolder && that);
     SessionHolder operator=(SessionHolder && that);
 
-    void Grab(const SessionHandle & sessionHandle)
-    {
-        Release();
-        mSession.SetValue(sessionHandle);
-    }
-
+    void Grab(const SessionHandle & sessionHandle) { mSession.SetValue(sessionHandle); }
     void Release() { mSession.ClearValue(); }
```

Comment:
These changes don't seem to be mentioned in the PR description.  What do they do?

Reply:
`Release` is `mSession.ClearValue`

It is unnecessary to call `ClearValue` before `SetValue`